core/tests/protractor_desktop/learnerDashboard.js

```diff
@@ -136,6 +136,46 @@ describe('Learner dashboard functionality', function() {
     users.logout();
   });
 
+  fit('should add exploration to play later list', function() {
```

Comment:
`fit` --> `it`.

I was expecting the lint test to catch "fit" keyword!

Reply:
Ah, sorry for that stupid mistake. Done!